portfolio/src/main/webapp/script.js

```diff
@@ -119,3 +119,21 @@ function togglePause() {
     paused = true;
   }
 }
+
+// function that fetches data from the /data URL
+// and displays it on the page
+function loadComments () {
+    fetch('/data').then(response => response.json()).then(comments => {
+        const commentList = document.getElementById('comments');
+        commentList.innerHTML = '';
+        commentList.appendChild(createListElement(comments[0]));
+        commentList.appendChild(createListElement(comments[1]));
+        commentList.appendChild(createListElement(comments[2]));
```

Comment:
instead of assuming there are always three comments, it is better to iterate the list and add all comments in there